fastly/resource_fastly_tls_subscription_test.go

```diff
@@ -22,14 +22,15 @@ func init() {
 func TestAccResourceFastlyTLSSubscription(t *testing.T) {
 	name := acctest.RandomWithPrefix(testResourcePrefix)
 	domain := fmt.Sprintf("%s.test", name)
+	domain2 := fmt.Sprintf("%sALT.test", name)
 
 	resourceName := "fastly_tls_subscription.subject"
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck:  func() { testAccPreCheck(t) },
 		Providers: testAccProviders,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccResourceFastlyTLSSubscriptionConfig(name, domain),
+				Config: testAccResourceFastlyTLSSubscriptionConfig(name, domain, domain2, domain),
```

Comment:
I'd prefer more descriptive variables to be passed into these functions (e.g. have them align with the actual signature defined). 

```suggestion
				Config: testAccResourceFastlyTLSSubscriptionConfig(name, domain1, domain2, commonName),
```

I appreciate that it's going to result in a semi-pointless variable assigned (e.g. `commonName = domain`) but I just feel it's much easier to understand what's going on at a glance without having to look at the terraform config function itself.

> Note: I presume we'd also have (based on my other comments): `commonName1 = domain1` and `commonName2 = domain2` as the next test step passes `domain2` as the 'common name'.